src/java.base/share/classes/java/nio/X-Buffer.java.template

```diff
@@ -1896,6 +1896,19 @@ public abstract sealed class $Type$Buffer
 
 #if[char]
 
+    /**
+     * {@inheritDoc}
```

Comment:
The method description here will need to start with  "Absolute bulk get method". This is important because CB defines both "absolute bulk get" and "relative bulk get methods", it has to be very clear in the API docs.

If the proposal goes again then I think the method description won't be inherited into CB, instead it will say that it transfers chars from this buffer into the given destination array.

Reply:
IIUC then you want me to replace `{@inheritedDoc}` by `This absolute bulk get method transfers chars from this buffer into the given destination array.`?

Reply:
Thank you, Alan. Fixed in https://github.com/openjdk/jdk/pull/21730/commits/29e1521fa25ba88dbbe4af077888044666f790f6.